gtsam/linear/GaussianFactorGraph.cpp

```diff
@@ -100,26 +100,34 @@ namespace gtsam {
   }
 
   /* ************************************************************************* */
-  vector<boost::tuple<size_t, size_t, double> > GaussianFactorGraph::sparseJacobian() const {
+  vector<boost::tuple<size_t, size_t, double> >
+  GaussianFactorGraph::sparseJacobian(
```

Comment:
Could we do this function (with its tests) in a separate PR? Then this PR can be exclusively about linear solvers.

Reply:
This change is in 837a09e, which is a late commit in Mandy's work on the Eigen backend. I can definitely separate them into different PRs, but I don't really see it as necessary because they will be rewritten in this PR anyway.